interpreter/src/key/internal.rs

```diff
@@ -27,8 +27,8 @@
  *
```

Comment:
Address the Missed mutations 
MISSED   interpreter/src/key/mod.rs:177:9: replace a_b_testing::string -> Result<(), Error> with Ok(()) in 1.5s build + 1.3s test
MISSED   interpreter/src/key/internal.rs:47:66: replace - with / in 1.4s build + 1.2s test
MISSED   interpreter/src/key/mod.rs:161:40: delete - in a_b_testing::build in 1.2s build + 1.3s test
MISSED   interpreter/src/key/mod.rs:161:49: replace + with - in a_b_testing::build in 1.2s build + 1.2s test
MISSED   interpreter/src/key/mod.rs:135:9: replace <impl Debug for Key>::fmt -> std::fmt::Result with Ok(Default::default()) in 1.4s build + 1.3s test
MISSED   interpreter/src/key/internal.rs:47:66: replace - with + in 1.4s build + 1.3s test
MISSED   interpreter/src/key/mod.rs:161:49: replace + with * in a_b_testing::build in 1.2s build + 1.2s test
MISSED   interpreter/src/key/mod.rs:162:45: replace + with - in a_b_testing::build in 1.2s build + 1.3s test
MISSED   interpreter/src/key/mod.rs:162:45: replace + with * in a_b_testing::build in 1.2s build + 1.3s test
MISSED   interpreter/src/key/mod.rs:162:36: delete - in a_b_testing::build in 1.1s build + 1.2s test
MISSED   interpreter/src/key/mod.rs:170:9: replace a_b_testing::extract -> Result<(), Error> with Ok(()) in 1.4s build + 1.3s test
MISSED   interpreter/src/key/internal.rs:245:23: replace > with == in Key::push in 1.1s build + 1.3s test
MISSED   interpreter/src/key/mod.rs:159:9: replace a_b_testing::build with () in 1.4s build + 1.3s test